docker/nginx/2.10/ckan.conf

```diff
@@ -0,0 +1,26 @@
+server {
+    listen       80;
+    listen [::]:80;
+    server_name  localhost;
+    access_log /var/log/nginx/access.log;
+    client_max_body_size 30M;
+
+    location / {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $http_host;
+        proxy_redirect off;
+        proxy_pass http://ckan:5000/;
+    }
+
+    location /csw {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $http_host;
+        proxy_redirect off;
+        proxy_pass http://ckan:8000/;
+    }
+
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+}
```

Comment:
Can this be removed? because we have the main ngnix conference

Reply:
yes, has been removed